### PR TITLE
"Text files should end with a newline character"

### PR DESCRIPTION
**Pull request by nilopc**
For the sake of completenes: https://insight.sensiolabs.com/projects/7a2c1cd1-5197-4822-8a4c-5ddaca48c769/analyses/10

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,5 @@ phpunit.xml
 composer.lock
 build
 vendor
-coverage.clover
\ No newline at end of file
+coverage.clover
+
```
Comment (whatthejeff) [inline .gitignore]: Looks like there's an extra space character.

Comment (nilopc) [inline .gitignore]: Yeah, that's what SensionLabs Insight is looking for. I know, it's silly.... using Fabien's php-cs-fixer gets this fixed too.

Comment (whatthejeff) [inline .gitignore]: Looks like you are ignoring files that only have a single space character (`U+0020`) in the name.

Comment (Ocramius) [inline .gitignore]: Please don't fix code that isn't broken.

The tool is simply wrong here, as it doesn't know how to correctly deal with `.gitignore`.

This is just a false positive.

**Pull request closed without merging**